bitnami/clickhouse/values.yaml

```diff
@@ -221,6 +228,12 @@ auth:
 ##
 logLevel: information
 
+## @section ClickHouse keeper configuration parameters
+## @param keeper.enabled Deploy ClickHouse keeper
```

Comment:
Can you modify the current text by "_Deploy ClickHouse keeper. Support is experimental._" and regenerate the README, please?

Reply:
ok